src/test/java/com/epam/tc/hw3/ex2/ExerciseTwoTest.java

```diff
@@ -0,0 +1,64 @@
+package com.epam.tc.hw3.ex2;
+
+import com.epam.tc.hw3.AbstractChromeTest;
+import com.epam.tc.hw3.PropertiesExtractor;
+import com.epam.tc.hw3.page.DifferentElementsPage;
+import com.epam.tc.hw3.page.DifferentElementsPageMainContainer;
+import com.epam.tc.hw3.page.IndexPage;
+import com.epam.tc.hw3.page.RightSideBar;
+import org.assertj.core.api.SoftAssertions;
+import org.testng.annotations.Test;
+
+public class ExerciseTwoTest extends AbstractChromeTest {
+
+    private String correctPageTitle = "Home Page";
+    private String correctUserName = "ROMAN IOVLEV";
```

Comment:
Username also could be moved and taken from resources

Reply:
Moved Username to the config.properties file